hasjob/templates/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Filename needs to be `manifest.json.jinja2`.

Reply:
Fixed